src/modm/board/nucleo_h723zg/board.hpp

```diff
@@ -113,14 +113,24 @@ struct SystemClock
 
 		Rcc::enableExternalClock(); // 8 MHz
 		const Rcc::PllFactors pllFactors1{
-			.range = Rcc::PllInputRange::MHz1_2,
+			.range = Rcc::PllInputRange::MHz4_8,
 			.pllM  = 4,		//   8 MHz / 4   =   2 MHz
```

Comment:
That change doesn't seem to be correct. The range refers to `ref_clk` which is the input to the `PLL` after the M divider (8 MHz / 4 = 2 MHz):
![Screenshot_20240104_013627](https://github.com/modm-io/modm/assets/25187160/0251b5bd-0747-4a38-8b05-3ca84d7bc04b)

Reply:
Whoops, fixed.